index.html

```diff
@@ -57,5 +57,232 @@
+        <div class="img-holder">
+          <img src="./images/Img Plaholder.png" alt="My Works">
+        </div>      
+      </div>
+
+      <div class="grid-item3">
+        <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="lang">
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavaScript</li>
+          <li>html</li>
+        </ul>
+        <button class="project" type="button"><a href="#">See Project</a></button>
+      </div>
+
+      <div class="grid-item2">
+        <div class="img-holder">
+          <img src="./images/Img Plaholder.png" alt="My Works">
+        </div>      
+      </div>
+
+      <div class="grid-item3">
+        <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="lang">
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavaScript</li>
+          <li>html</li>
+        </ul>
+        <button class="project" type="button"><a href="#">See Project</a></button>
+      </div>
+    </section>
+
+    <!-- work section end  -->
+
+			<!-- About me section start -->
+			<section>
+				<div class="grid-item1">
+					<h2 class="headline">About me</h2>
+					<div class="about">
+						<p>I can help you bulid a product, feature or website. Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+					</div>
+				</div>
+				<div class="grid-item3">
+					<button class="resume" type="button"><a href="#">Get My Resume</a></button>
+			</section>
+	
+				<!-- About section end -->
+
+      <!-- Lunguage section start -->
+
+			<section class="lang-section">
+				<div class="background">
+					<div class="grid-item3">
+						<div class="lang-icon">
+							<img src="./images/Icon Langguage.png" alt="Lunguage icon">
+						</div>
+						<h3 class="Language">Languages</h3>
+						<ul class="lang2">
+							<li>JavaScript</li>
+							<li>Ruby</li>
+							<li>Html</li>
+							<li>css</li>
+						</ul>
+					</div>
+				</div>
+			</section>
+	
+			<!-- Lunguage section end -->
+
+      <!-- Framwork section start -->
+
+			<section class="frameworks">
+				<div class="frame-icon">
+					<img src="./images/Framwork-icon.png" alt="Frameworks icon">
+				</div>
+				<h3 class="frame-title">Frameworks</h3>
+				<ul class="frame-list">
+					<li>Bootstrap</li>
+					<li>Ruby on rails</li>
+					<li>RSpec</li>
+					<li>Capybara</li>
+					<li>Selenium</li>
+				</ul>
+			</section>
+	
+			<!-- Framwork section end -->
+
+    <!-- Skills section starts -->
+    
+    <section class="skills">
+      <div class="skills-icon">
+        <img src="./images/Skills-icon.svg" alt="Skills icon">
+      </div>
+      <h3 class="Skills-title">Skills</h3>
+      <ul class="skills-list">
+        <li>Codekit</li>
+        <li>GitHub</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </section>
```

Comment:
- [ ] The languages, frameworks and skills shouldn't be in their own section tags rather they should be inside of the "About me" section. Kindly fix that. 🙏🏾